torch/quantization/fx/quantization_patterns.py

```diff
@@ -1690,3 +1689,309 @@ def convert(self,
         setattr(modules[parent_name], name, quantized_standalone_module)
         modules[str(node.target)] = quantized_standalone_module
         return quantized_graph.node_copy(node, load_arg(quantized=input_quantized_idxs))
+
+
+class ConvReLUQuantizeHandlerNew(QuantizeHandler):
```

Comment:
nit: would it be possible to put these in a separate file while the structure is temporary, so it's easier for other people to understand the rest of `quantization_patterns.py`?

Reply:
sure, can I land first and follow up later, planning to do migration asap after this PR, looks like it takes time to import things for these handlers in the new file.